examples/extend-graphql-schema-nexus/schema.ts

```diff
@@ -28,3 +32,63 @@ export const lists = {
     },
   }),
 };
+
+export function extendGraphqlSchema(baseSchema: GraphQLSchema) {
+  const NexusPostQuery = nexus.extendType({
+    type: 'Query',
+    definition(t) {
+      t.field('nexusPosts', {
+        type: nexus.nonNull(nexus.list('Post')),
+        args: {
+          authorId: nexus.stringArg(),
+          seconds: nexus.nonNull(nexus.intArg({ default: 600 })),
+        },
+        async resolve(root, { authorId, seconds }, context: Context) {
+          const cutoff = new Date(Date.now() - seconds * 1000);
+
+          return await context.db.Post.findMany({
+            where: {
+              ...(authorId ? { author: { id: authorId } } : null),
+              publishDate: { gt: cutoff },
+            },
+          });
+        },
+      });
+    },
+  });
+
+  const NexusThing = nexus.objectType({
+    name: 'NexusThing',
+    definition(t) {
+      t.int('id');
+      t.string('title');
+    },
+  });
+
+  const NexusThingQuery = nexus.extendType({
+    type: 'Query',
+    definition(t) {
+      t.nonNull.list.field('things', {
+        type: NexusThing,
+        resolve() {
+          return [
+            { id: 1, title: 'Keystone' },
+            { id: 2, title: 'Prisma' },
+            { id: 3, title: 'Nexus' },
+          ];
+        },
+      });
+    },
+  });
+
+  return nexus.makeSchema({
+    mergeSchema: {
+      schema: baseSchema,
+    },
+    types: {
+      NexusThing,
+      NexusPostQuery,
+      NexusThingQuery,
+    },
+  });
+}
```

Comment:
I think this example shows that `nexus` is ~equivalent to our `graphql-ts` example?